apollo-federation/cli/src/main.rs

```diff
@@ -17,6 +18,32 @@ use clap::Parser;
 mod bench;
 use bench::run_bench;
 
+#[derive(Parser)]
+struct QueryPlannerArgs {
+    /// Enable @defer support.
+    #[arg(long, default_value_t = false)]
+    enable_defer: bool,
+    /// Turn off fragment reuse in subgraph queries.
+    #[arg(long = "no-reuse-fragments", default_value_t = true, action = clap::ArgAction::SetFalse)]
```

Comment:
Wouldn't it be more consistent with harness to have "reuse-fragments" with false as default?

Reply:
my goal here is to match the default configuration of the query planner.
I would've preferred to have `--reuse-fragments` to explicitly turn it on and `--no-reuse-fragments` to explicitly turn it off, and using the QueryPlannerConfig default if neither is given. Maybe that's possible if i make it an Option, actually 🤔

Reply:
I had a similar discussion with @lrlna over at performance-harness. I thought the same thing and added `--disable-reuse-fragments`, but I realized it would be another complexity added, while `--reuse-fragments <true/false>` option is used everywhere else.